app/apps/server/bridges/livechat.js

```diff
@@ -72,14 +72,21 @@ export class AppLivechatBridge {
 		return this.orch.getConverters().get('rooms').convertRoom(result.room);
 	}
 
-	async closeRoom(room, comment, appId) {
+	async closeRoom(room, comment, closer, appId) {
 		this.orch.debugLog(`The App ${ appId } is closing a livechat room.`);
 
-		return Livechat.closeRoom({
-			visitor: this.orch.getConverters().get('visitors').convertAppVisitor(room.visitor),
+		const closeData = {
 			room: this.orch.getConverters().get('rooms').convertAppRoom(room),
 			comment,
-		});
+		};
+
+		if (closer?.id) {
```

Comment:
Hey @renatobecker! Can you give me some help? 
I'm not really familiar with the Livechat codebase. Do you think this is the right approach in this situation, sending the `visitor` and `user` in different situations?